src/ActiveCell.js

```diff
@@ -18,7 +18,8 @@ type Props<Cell, Value> = {|
   hidden: boolean,
   mode: Types.Mode,
   edit: () => void,
-  getBindingsForCell: Types.getBindingsForCell<Cell>
+  getBindingsForCell: Types.getBindingsForCell<Cell>,
+  setCellCommit: ({ before: Cell | null, after: Cell | null }) => void
 |};
 
 class ActiveCell<Cell, Value> extends Component<Props<Cell, Value>> {
```

Comment:
Try implement commit here.
Add documentation for scenario:
```
1. When active cell has changed and the previous cell was edited
2. When active cell exited edit and was edited
```

Reply:
@iddan 
I've tried  a few stuff but found that its not possible to do it with `<ActiveCell/>`.

As long as we want to create the new event `onCellCommit` it must be triggered under some kind of **componentWillUnmount**. 

ActiveCell never unmounts, and triggering a change on each **componentDidUpdate** will lead us to an `onCellChange` **instead of** `onCellCommit`. 

So unless we want to change `<ActiveCell/>`'s behavior, our option is to do it under `<DataEditor/>`'s  **componentWillUnmount**.